device-catalog.html

```diff
@@ -9,186 +9,224 @@
+                                <p>1500 руб.</p>
+                            </span>                            
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>
+                        </li>
+                        <li class="content-item">
+                            <img src=".//img/item-3.jpg" alt="Непотопляемая селфи-палка" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Непотопляемая селфи-палка</h3></a>
+                                <p>2500 руб.</p>
+                            </span>
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>                        
+                        </li>
+                        <li class="content-item">
+                            <img src=".//img/item-4.jpg" alt="Селфи-палка «Следуй за мной»" max-width="360" max-height="380">
+                            <span>
+                                <a href="#"><h3>Селфи-палка «Следуй за мной»</h3></a>
+                                <p>4900 руб.</p>
+                            </span>
+                            <div class="content-buttons">
+                                <button type="button">в корзину</button>
+                                <button type="button">Добавить к сравнению</button>
+                            </div>                        
+                        </li>
+                    </ul>
+    
+                </div>
+                <div class="content-pages">
+                    <a class="page-back" href="#">назад</a>
+                    <ul>
+                        <li>1</li>
+                        <li><a href="#">2</a></li>
+                        <li><a href="#">3</a></li>
+                    </ul>
+                    <a class="page-forward" href="#">вперед</a>
+                </div>
+                <aside class="sibe-bar">
+                    <h2>Фильтр:</h2>
+                    <form class="filter" method="get" action="#">
+                        <fieldset class="filter-price">
+                            <legend>Стоимость</legend>
+                            <input type="range" name="price" id="filter-price-from" min="0" step="500" value="5000">
+                            <input type="range" name="price" id="filter-price-to" max="10000" step="500" value="5000">
+                            <!-- С этим у меня пока проблема -->
+                        </fieldset>
+                        <fieldset class="filter-color">
+                            <legend>Цвет</legend>
+                            <ul>
```

Comment:
imo, no need in ul when you use grouping tag fieldset. but your version is OK as well

Reply:
maybe i will need this structure in css. If not I remove ul
https://medium.com/claritydesignsystem/pure-css-accessible-checkboxes-and-radios-buttons-54063e759bb3#.c7bogvf04